src/main/java/com/exasol/releasedroid/usecases/Repository.java

```diff
@@ -83,5 +84,12 @@
      * 
      * @return full name as a String
      */
-    public String getFullName();
+    public String getName();
+
+    /**
+     * Get a parsed Maven pom file.
+     *
+     * @return instance of {@link MavenPom}
+     */
+    public MavenPom getMavenPom();
```

Comment:
if I understand, this should not be here at all, unless we definitely say "hey, the Release Droid will only work for maven repositories and we have no plans at all to change that".

Have in mind that your `usecases` packages should know nothing about any external dependencies, unless you really want to bind yourself to it (which will make it harder if in the future you want to support non-maven repositories).

Reply:
I'm going to make the repository interfaces structure more complicated when I need it (when we add a project different from Maven). I don't want to make it complicated right now, it only confuses me and reviewers